website_team/controllers/main.py

```diff
@@ -37,6 +27,5 @@ def user(self, user_gh):
             return http.request.render('website_team.test_user', {
                 'user': current_user,
                 'Youtube_link': youtube_id,
-                # 'alias': aliasq
+                'AliasEmail': alias_email_full
```

Comment:
I recommend to use lower case variable names. Use CamalCase only for class names

                  'youtube_link': youtube_id, 
               'alias_email': alias_email_full